### ISSUE TITLE
Inference sample code/script

### ISSUE THREAD

**Issue by ztqwerty** [Koushik0901/Swift-SRGAN#2]
Hi, 

Can you please provide an example for running inference with a pre-trained model?

Thanks,
Tony

**Comment by ARTUROSING** [comment]
Yes, I would also like to see a demo, and an example of inference models.
The technology is quite unique!
You can adapt Swift-SRGAN for Real-ESRGAN to work in real time with impressive speed!
